src/class-rest.php

```diff
@@ -0,0 +1,103 @@
+<?php
+/**
+ * REST API class
+ *
+ * @package Parsely
+ * @since 3.1.0
+ */
+
+declare(strict_types=1);
+
+namespace Parsely;
+
+use WP_Post;
+
+/**
+ * Injects Parse.ly Metadata to WordPress REST API
+ *
+ * @since 3.1.0
+ */
+class Rest {
+	private const PARSELY_REST_VERSION    = '1.0.0';
+	private const PARSELY_REST_FIELD_NAME = 'parsely';
```

Comment:
```suggestion
	private const REST_VERSION    = '1.0.0';
	private const REST_FIELD_NAME = 'parsely';
```

If we're more specific about what this class is doing, I don't think we need these prefixes.